src/app/shared/components/number-plate/number-plate.component.spec.ts

```diff
@@ -24,12 +24,12 @@ describe('NumberPlateComponent', () => {
 
   it('should format a standard vrm', () => {
     component.vrm = 'AA21AAA';
-    expect(component.vrm).toEqual('AA21 AAA');
+    expect(component.vrm).toBe('AA21 AAA');
```

Comment:
not 100% how they work in js though

Reply:
so you're right, if it was comparing the entire object it causes a fail because of memory location, but because it's just comparing the one string its alright.

Reply:
Also for context Tom, the linter basically now forces you to use the correct one based on what it is you're comparing it against hence why we've changed so many to "toBe"